docs/Makefile

```diff
@@ -25,6 +25,12 @@ help:
 %: Makefile .venv
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
+# Make the site's html in docker
+docker_html:
+	mkdir -p _build
+	docker build -t tarmakdocs:latest .
+	docker run -it -v "$$(pwd):/site" -u `stat -c "%u:%g" _build` -w /site tarmakdocs:latest
```

Comment:
I would suggest using `docker cp` instead. It is more compatible (mac) than volume mounts. Also doesn't create root owned files.

```
docker create
docker cp docs into
docker start
docker cp build out of it
```

Reply:
Ahh yeah, that's nicer. Hopefully 7df1d75dc7d6f3c0659bb4b47dc02f88c580692d is looking better.